src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,45 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin onTablet {
+  @media (min-width: 744px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width: 1260px) {
+    @content;
+  }
+}
+
+@mixin headerNavigationGrid($repeatedItem, $repeatedWidth, $gridGap) {
+  display: grid;
+  align-items: center;
+  grid-template-columns: 1fr repeat($repeatedItem, $repeatedWidth);
+  gap: $gridGap;
+}
+
+%bgImageNormalize {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
```

Comment:
It's a good practice to add a comment explaining the purpose of a placeholder (e.g., '/* Normalizes the background image properties */').